app/components/List.jsx

```diff
@@ -4,8 +4,19 @@ import React from 'react';
 import ListItem from './ListItem';
 import ReactCSSTransitionGroup from './TimeoutTransitionGroup';
 import { mergeAndPrefix } from '../utils/stylePropable';
+import { List } from 'material-ui';
+
+export default class extends React.Component {
+  static propTypes = {
```

Comment:
Just name the class something different then. The component itself can still be named `List`.